src/mpid/ch4/netmod/ofi/ofi_impl.h

```diff
@@ -119,6 +119,27 @@ int MPIDI_OFI_handle_cq_error(int vci, int nic, ssize_t ret);
         } \
     } while (0)
 
+#define MPIDI_OFI_CALL_RETRY_AM(FUNC,vci_,STR)                          \
+    do {                                                                \
+        ssize_t _ret;                                                   \
+        do {                                                            \
+            _ret = FUNC;                                                \
+            if (likely(_ret==0)) break;                                  \
+            MPIDI_OFI_ERR(_ret != -FI_EAGAIN,                  \
```

Comment:
commit message is `remove MPIDI_OFI_ERR` but I think it's still in use?

Reply:
I'll update the commit message

Reply:
Updated to: "ch4/ofi: remove MPIDI_OFI_ERR usages outside ofi_impl.h".